syne_tune/optimizer/baselines.py

```diff
@@ -778,6 +779,50 @@ def __init__(
         )
 
 
+class NSGA2(FIFOScheduler):
+    """
+
+    See :class:`~syne_tune.optimizer.schedulers.searchers.RandomSearcher`
+    for ``kwargs["search_options"]`` parameters.
+
+    :param config_space: Configuration space for evaluation function
+    :param metric: Name of metric to optimize
+    :param population_size: See
+        :class:`~syne_tune.optimizer.schedulers.searchers.RegularizedEvolution`.
+        Defaults to 100
+    :param sample_size: See
+        :class:`~syne_tune.optimizer.schedulers.searchers.RegularizedEvolution`.
+        Defaults to 10
+    :param random_seed: Random seed, optional
+    :param kwargs: Additional arguments to
+        :class:`~syne_tune.optimizer.schedulers.FIFOScheduler`
+    """
+
+    def __init__(
+        self,
+        config_space: Dict[str, Any],
+        metric: List[str],
+        mode: Union[List[str], str] = "min",
+        population_size: int = 100,
```

Comment:
Regarding this default, the value seems high to me. Do you think a smaller value like 20 would work better on "average"? (for instance if we were to run in our benchmarks I suppose 20 might be better?)

Reply:
Good question. I followed pymoo's default, but it indeed might be a bit high for our benchmarks. I'll set it to 20